yarn-project/ethereum/src/utils.ts

```diff
@@ -168,28 +168,6 @@ export function formatViemError(error: any, abi: Abi = ErrorsAbi): FormattedViem
     const errorClone = structuredClone(error);
 
     // Helper function to recursively remove ABI properties
-    const stripAbis = (obj: any) => {
```

Comment:
Just moving these huge anon declared functions to be actual functions as they make the function unreadable when they are all inline